opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/config/CollectionConfigurationResource.java

```diff
@@ -67,11 +65,7 @@ public Response getCollectdConfigurationForLocation(@PathParam("location") final
             return Response.status(Response.Status.NOT_FOUND).build();
         }
 
-        final List<String> collectionPackageNames = def.getCollectionPackageNames();
-        if (collectionPackageNames != null && collectionPackageNames.size() > 0) {
-            final CollectdConfiguration collectdConfig = m_collectdConfigResource.get().getCollectdConfigurationForPackages(collectionPackageNames);
-            return Response.ok(collectdConfig).build();
-        }
+        // TODO: Patrick: do we still need this method?
```

Comment:
@fooker we should discuss this.

Reply:
I think this is a leftover from the first days minion work - the whole component can go away.

Reply:
:+1: I removed it